deploy/crds/examples/mobile-security-service_v1alpha1_mobilesecurityserviceapp_cr.yaml

```
@@ -1,14 +1,16 @@
-# Example of a CR to create/apply and allow the operator manage the applications.
-# Note that a similar YAML file should be generated for each app which will be created/manage by the Mobile Security Service
+# ---------------------------------
+# Example of App CR
+# ---------------------------------
+
+# Usage: Shows how to define the object to create/update the applications in the service side
 apiVersion: mobile-security-service.aerogear.com/v1alpha1
 kind: MobileSecurityServiceApp
 metadata:
   name: mobile-security-service-app
 spec:
-  # The appName spec defines the appName of the app in the Service side, it also is used to create the ConfigMap name
-  # NOTE: To update the name just update this spec and re-apply the CR
+  # The appName should be the name of your mobile application. It will be used to create a new app in the service, the labels and the name of the ConfigMap.
```

```suggestion
  # The appName is the name of your mobile application. It is used to create a new app in the service, the labels and the name of the ConfigMap.
```

I believe the present tense (ie `is` not `will`) is preferred by the docs team, thus the suggestion (know its our repo readme and not published docs, but convention to use present tense seems to read well in docs I feel)

---

ok.